librad/src/meta/profile.rs

```diff
@@ -64,23 +64,93 @@ pub enum ProfileImage {
     Url(Url),
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[derive(Deserialize, Serialize, Debug, Clone, Copy, PartialEq, Eq)]
+pub struct GeoArc {
```

Comment:
I was looking at this resource: https://astro.unl.edu/naap/motion1/tc_units.html

+/- could mean North and South or East and West. Something I found interesting is that the degrees max for N/S is 90 and for E/W is 180. 

It might make sense to have two separate types here one for latitude and one for longitude, `deg`, `min`, and `sec` could be all be `u8`, and we carry an `enum NS` or `enum EW` indicating the direction. All the while ensuring the invariants of those values above via smart constructors.